docs/walkthrough/create-training-data.md

```diff
@@ -62,10 +58,27 @@ At the model saving time, you will discover, we are saving two models to your lo
 ````
 
 
+````{tab} two elements per row
+If you want two elements to be semantically close together, they can be placed on the same row as a pair, doing so will assign each pair a distinct label:
+
+```markdown
+This is an English sentence         Das ist ein englischer Satz
+This is another English sentence    Dies ist ein weiterer englischer Satz
+...
+```
+
+```{warning}
+When reading in files constructed in this way, a unique label is generated for each pair. This means that each class in the dataset will only contain a single pair of elements, resulting in some sampling methods that require more than two elements per class not working.
```

Comment:
"When reading in files constructed in this way, a unique label is generated for each pair. Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements."